tests/test_build.py

```diff
@@ -179,6 +181,33 @@ def test_no_base_image(tmpdir):
     assert "no base image specified" in str(exc.value)
 
 
+def test_copy_from_is_blocked(tmpdir):
+    """test when user has specified COPY --from=image (instead of builder)"""
+    dfp = df_parser(str(tmpdir))
+    if MOCK:
+        mock_docker()
+    source = {'provider': 'path', 'uri': 'file://' + str(tmpdir), 'tmpdir': str(tmpdir)}
+
+    dfp.content = dedent("""\
+        FROM monty AS vikings
+        FROM python
+        COPY --from=vikings /spam/eggs /bin/eggs
+        COPY --from=0 /spam/eggs /bin/eggs
+        COPY src dest
+    """)
+    InsideBuilder(get_source_instance_for(source), 'built-img')  # calls set_df_path at init
```

Comment:
stylistic nit: comments should proceed code, not caption them.
also, not sure the inline comment is necessary or even helpful.